src/features/appearance/index.ts

```diff
@@ -247,7 +249,17 @@ function generateServiceRibbonWidthStyle(
       margin-top: -${width}px !important;
     }
     .darwin .sidebar {
-      height: ${width + sidebarSizeBias}px !important;
+      height: ${width + verticalStyleOffset - 3 - sizeDragArea}px !important;
+    }
+    .darwin .app .app__content {
+      padding-top: ${
+        width + sidebarSizeBias + (sizeDragArea !== 0 ? 4 - sizeDragArea : 4)
+      }px !important;
+    }
+    .darwin .workspaces-drawer {
+      margin-top: -${
+        width + verticalStyleOffset - 5 - sizeDragArea
+      }px !important;
```

Comment:
This might be weird but it's the pixels it needs to align properly. In the future, we should fully move to a fully functional flexbox implementation for this.